skbio/alignment/_pairwise.py

```diff
@@ -795,9 +838,8 @@ def _init_matrices_nw_no_terminal_gap_penalty(
 
 
 def _compute_substitution_score(aln1_chars, aln2_chars, substitution_matrix,
```

Comment:
Nevermind, the order of the helpers was confusing me, the diff made it look recursive on it's own.